### DISCUSSION TITLE
Error pushing to branch

### DISCUSSION THREAD

**Discussion by tiagonmas** [banco-alimentar/alimentestaideia.pt#159] [Q&A]
@guerrerotook  any idea why @joalmeid  is getting this error ?

! [remote rejected] feature/aspnetcore-new-website-nuget -> feature/aspnetcore-new-website-nuget (protected branch hook declined)
error: failed to push some refs to 'https://github.com/banco-alimentar/alimentestaideia.pt.git'

Remote: error: GH006: Protected branch update failed for refs/heads/feature/aspnetcore-new-website-nuget.
Remote: error: Required status check "Alimentestaideia.pt core" is expected. At least 2 approving reviews are required by reviewers with write access.

**Comment by tiagonmas** [comment]
ah, now I remember. @joalmeid  you need to have 2 approvers :) as the error message states. can you add me and luis ?

**Reply by tiagonmas** [reply]
wait ... this is not a PR, you are just trying to push to your own repo ... hum

**Reply by tiagonmas** [reply]
current rule requires all PRs to  branches feature/aspnetcore-* to have 2 reviews

**Reply by tiagonmas** [reply]
also:
![image](https://user-images.githubusercontent.com/42462/112990947-83419a00-915e-11eb-84a9-73b6075344f0.png)

**Comment by joalmeid** [comment]
We need approvers for  feature branches? . That would make sense when I do a PR, not a push. IMO
I just read you got to the same conclusion

**Reply by guerrerotook** [reply]
Are you pushing to your own branch?

**Comment by guerrerotook** [comment]
What it says is that you need to pass the build pipeline, Alimentestaideia.pt core and two approvers.
